### ISSUE TITLE
Error loading keyboard applet after upgrading to from 19 to 19.1

### ISSUE THREAD

**Issue by vlad0337187** [linuxmint/cinnamon#8373]
```
 * Cinnamon version (cinnamon --version): 4.0.9
 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...): Mint 19.1
 * Graphics hardware *and* driver used: Intel Corporation HD Graphics 5500 + nvidia-driver-390; Nvidia GT840m
 *  64 bit
 * Attach /home/<username>/.xsession-errors, or /var/log/syslog
[xsession-errors.txt](https://github.com/linuxmint/Cinnamon/files/2839765/xsession-errors.txt)
[syslog.txt](https://github.com/linuxmint/Cinnamon/files/2839768/syslog.txt)

* system info: https://termbin.com/2xt8
 ```

**Issue**
Doesn't see Keyboard widget on panel.

**Steps to reproduce**
Install Mint 19, upgrade it to 19.1; reboot.
See no keyboard applet on panel.

**Expected behaviour**
See keyboard applet on panel.

**Other information**
Video: https://www.youtube.com/watch?v=xQqEDu2RC4s

Sometimes it appears and disappears (just blinks) after changing keyboard settings (see video).
Applets manager displays it with (-) sign, meaning there is an error in plugin.
If to reload it's code from looking glass - applet manager shows all is ok (made before recording video), but keyboard widget is still hidden.

**Comment by smurphos** [comment]
You've not provided a link for your video. What keyboard settings are you changing to get it to display? By design it should only display if there are 2 or more installed keyboard layouts.

**Comment by vlad1777d** [comment]
@smurphos , sorry, was tired when wrote that, didn't noticed.
Updated link to video.

**Comment by vlad1777d** [comment]
Recently did another update, after first reboot, and after all next reboots plugin was loaded successfully.

(there were several packages, remember only `mesa` between them)
(maybe I did another things, but didn't noticed what exactly affected this, because I reboot only after updates installed)
(also I reinstalled cinnamon-common, don't know, maybe this affected problem. Don't remember, did I made reboot after that reinstall, maybe I reinstalled cinnamon-common, made some actions, rebooted PC, after it applet started working...)

Think when I'll have time to install 19 to Virtualbox, upgrade it to 19.1 and check this issue.
